src/EthMultiVault.sol

```diff
@@ -433,6 +448,7 @@ contract EthMultiVault is IEthMultiVault, Initializable, ReentrancyGuardUpgradea
         timelocks[opHash].executed = true;
 
         emit ExitFeeSet(id, exitFee, oldExitFee);
+        emit OperationExecuted(SET_EXIT_FEE, data);
```

Comment:
3.17 - Issue an Event for Operation Execution

Here, we are now emitting an event for `OperationExecuted`